modules/post/windows/manage/reflective_dll_inject.rb

```diff
@@ -6,45 +6,187 @@
 require 'msf/core/post/windows/reflective_dll_injection'
 
 class MetasploitModule < Msf::Post
+  include Msf::Post::File
+  include Msf::Post::Windows::Priv
+  include Msf::Post::Windows::Process
   include Msf::Post::Windows::ReflectiveDLLInjection
 
-  def initialize(info={})
-    super( update_info( info,
-      'Name'          => 'Windows Manage Reflective DLL Injection Module',
-      'Description'   => %q{
-        This module will inject into the memory of a process a specified Reflective DLL.
-      },
-      'License'      => MSF_LICENSE,
-      'Author'       => [ 'Ben Campbell'],
-      'Platform'     => [ 'win' ],
-      'SessionTypes' => [ 'meterpreter' ],
+  def initialize(info = {})
+    super(update_info(info,
+      'Name' => 'Windows Manage Reflective DLL Injection
+      Module with arguments',
+      'Description' => '
+      This module will inject a specified reflective DLL
+      into the memory of a process, new or existing,
+      passing any arguments.
+      ',
+      'License' => MSF_LICENSE,
+      'Author' => 'Ben Campbell,b4rtik',
```

Comment:
This should be an array of strings.

```suggestion
      'Author'  => [ 'Ben Campbell', 'b4rtik' ],
```